pkg/cli/cmd/authorizer/authorizer.go

```diff
@@ -0,0 +1,10 @@
+package authorizer
+
+type AuthorizerCmd struct {
+	EvalDecision EvalDecisionCmd `cmd:"" help:"evaluate policy decision" group:"authorizer"`
```

Comment:
In the `topaz test` we refer to Authorizer Is calls as `check decision`, might be good to unify terminology.